### ISSUE TITLE
Setting CCLabel.IsAntialiased = false throws NullReferenceException

### ISSUE THREAD

**Issue by vchelaru** [mono/CocosSharp#213]
The following code throws a NRE:

```
label = new CCLabel("Test", "fonts/alphbeta.ttf", 22, CCLabelFormat.SystemFont);
this.AddChild (label);
label.IsAntialiased = false;
```

I expect to be able to use this so that point texture filtering is applied so my fonts can have a pixellated look.

Example of fonts without this value applied:

![noantialiasoncclabel](https://cloud.githubusercontent.com/assets/729631/7730123/aec798cc-fed5-11e4-9245-73d959b8588d.PNG)

**Comment by kjpou1** [comment]
@vchelaru This should be fixed already but after 1.4.0.0 release https://github.com/mono/CocosSharp/commit/b96a728d801ca95332ab03b36d7bcd919e427cc9

**Comment by vchelaru** [comment]
@kjpou1 Thanks!  For reference, duplicate issue here:  https://github.com/mono/CocosSharp/issues/173
